app/controllers/claims/support/schools_controller.rb

```diff
@@ -6,4 +6,44 @@ def index
   def show
     @school = Claims::School.find(params.require(:id))
   end
+
+  def new
+    @school = Claims::School.new
+  end
+
+  def check
+    if school.valid? && !school.claims?
+      @school = school.decorate
+    else
+      school.errors.add(:urn, :taken) if school.claims?
+      render :new
+    end
+  end
+
+  def create
+    school.claims = true
+    if school.save
```

Comment:
In that case, can we compact these two lines?

```suggestion
    if school.update(claims: true)
```